File: JavaScript/function.js
/* 
* 
* Q1: Write JavaScript to store and retrieve a JSON file on your local 
* storage with the following information:
* 
* Name: John Doe
* Age: 35
* City: Collingwood
* 
*/

// person object
var person = { 'Name': 'John Doe', 'Age': 35, 'City': 'Collingwood' };

// assuming we're storing on browser local storage
localStorage.setItem('person', JSON.stringify(person));

// retreiving person object from local storage
var LocalPerson = localStorage.getItem('person');

// printing to console that I have done the task
console.log('Stored Data: ', JSON.parse(LocalPerson));

File: Python/function.py
# Q2: Write Python code to create a dictionary for the same 
# information above and convert it to JSON

# importing JSON package
import json 

name = 'John Doe'
age = 35
city = 'Collingwood'

personDict = {
    "Name": name,
    "Age": age,
    "City": city,
}

# Converting into JSON
localPerson = json.dumps(personDict)

# Printing JSON as string
print(localPerson)


